integration_tests/src/main/python/fastparquet_utils.py

```diff
@@ -0,0 +1,126 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+
+from pyspark.sql import Row
+
+def get_fastparquet_result_converter():
```

Comment:
The name may be misleading. It's not returning the convertor but actual the converted result.

Not sure how we plan to use this. Probably you was meaning ```get_fastparquet_canonicalized_result```.